Sources/TestSupportExecutable/main.swift

```diff
@@ -0,0 +1,56 @@
+import Basic
+import POSIX
+import libc
```

Comment:
... and I'd probably make the sole existing command a subcommand "<exec> lock-test <arg1> <arg2> <arg>3" so it is obvious how to extend.